src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -2376,6 +2376,27 @@ public enum ConfigNodes {
 			"5",
 			"",
 			"#How many hours an attacking outlaw will be jailed for."),
+	JAIL_POW_JAIL_HOURS(
+		"jail.pow_jail_hours",
+		"5",
+		"",
+		"#How many hours an attacking enemy will be jailed for."),
+	JAIL_MAXSET_JAIL_HOURS(
+		"jail.maxset_jail_hours",
+		"5",
+		"",
+		"#The maximum hours that a mayor can set when jailing someone, full number expected."),
+	JAIL_FEE_INITIAL_AMOUNT(
+		"jail.fee_initial_amount",
+		"-1",
+		"",
+		"#Amount that it costs per player jailed. Set to -1 to disable"),

```

Comment:
Comments should clarify that it is the town bank paying this cost.

Reply:
ConfigNodes.java improved to clarify money is withdrawn from Town bank, effect also changed for jail upkeep fee.